### PR TITLE
Added Wishlist And Cart Pages

### PR DESCRIPTION
**Pull request by SimranjitKaurMaan**
image
image

### CODE REVIEW

**Review by rahulcs754 — APPROVED** [review]
I look at your code that is clear and nice.
and  design of  wishlist is good

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                        </li>
+                    </ul>
+                </nav>
+                <a href="#" class="user">
+                  <img class="img-sm-2" src="./assets/user.svg" alt="user"/>
+                </a>
+            </div>
+        </header>
+        <main>
+          <div class="main-section">
```
Comment (sadathshariff) [inline cart.html]: Just a suggestion you can use section tag here

Comment (SimranjitKaurMaan) [reply cart.html]: Got it. I have addressed this.

File: stylesheets/styles.css [hunk 2]
```diff
@@ -208,6 +229,33 @@ a.user {
...
+  "main main"
+  "main main"
+  "footer footer";
+}
+
+.wishlist-wrapper {
+  display: grid;
+  width: 100%;
+  grid-gap: 1.5rem;
+  grid-template-columns: 13rem 13rem 13rem 13rem 1fr;
```
Comment (sadathshariff) [inline stylesheets/styles.css]: you can use repeat here just a suggestion

**Review by sadathshariff — APPROVED** [review]
The code looks clean and the class names as well.

File: stylesheets/styles.css (hunk 2 above)
Comment (SimranjitKaurMaan) [reply stylesheets/styles.css]: Thanks. I will replaced with repeat.

**Comment by SimranjitKaurMaan** [comment]
> The code looks clean and the class names as well.

Thanks for reviewing.

**Pull request merged**